db/migrate/20200909091201_create_users.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
+      t.string :user_name
+      t.string :email
+      t.integer :password
+
+      t.timestamps
+    end
+    add_index :users, :password, unique: true
```

Comment:
### password
- [ ] Should be a string, not an integer
- [ ] Should not be unique, otherwise, no user can have a repeated password. this can be a very bad user experience